ls1intum/Artemis
# `Exam mode`: Fix breadcrumbs for grading keys

### Checklist
#### General
#### Client

### Motivation and Context
The current breadcrumbs for the grading keys have two issues:
- There is a link to "overview" which does not correspond to any side -> clicking that spawns an error messsage
- There is no translation for the page name

### Description


### Steps for Testing
Prerequisites:
- 2 Student
- 1 Course with Grade Key active

1. Navigate to the grade key
2. Check the breadcrumb navigation

#### Exam Mode Testing

Prerequisites:
- 1 Student
- 1 Exam with a Grade Key and results are already published

1. View your grade on the overview page.
2. Open the grade key
3. Check the breadcrumb navigation

### Review Progress
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Screenshots
old:
![grafik](https://user-images.githubusercontent.com/26540346/219772869-46d3f2fc-da55-4a4d-9ed4-7b2254990b07.png)

new:
![grafik](https://user-images.githubusercontent.com/26540346/219772847-ccc3f67c-26f7-4880-8993-a60ade18e9c9.png)
